src/services/users/index.js

```diff
@@ -49,7 +50,7 @@ const createNewUser = async (user) => {
 	const body = { id: userProfile.id, email: userProfile.email }
 	const accessToken =  generateAccessToken({
 		data: {user : body}, 
-		expiry : '15m'
+		expiry : '1m'
```

Comment:
@beblicarl kindly do remember to revert this back to what it was

Reply:
done